Pipfile

```diff
@@ -105,6 +104,10 @@ python-dotenv = "==1.0.0"
 python-jose = "==3.3.0"
 setuptools = "==67.2.0"
 raven = "==6.10.0"
+frictionless = {editable = true, ref = "jinja2_2_10_1", git = "https://github.com/fjelltopp/frictionless-py", extras=["ckan"]}
+frictionless-geojson = {editable = true, ref = "v1.1.0rc1", git = "https://github.com/fjelltopp/frictionless-geojson"}
+jinja2 = "==2.10.1"
+tableschema = "*"
```

Comment:
Please change "*" dependency to a more specific one.
We also accept `~=` to allow patch versions updates. I don't think we should go with `>=` as it might create troubles by accident major dependencies upgrades.

Reply:
So I don't know what tableschema version we should lock to as there's no version in the ckanext-validation requirements, should I just pin to whatever the lockfile has picked?

And "oops" on ">="; in package.json ">=" is what "~=" is in Pipfile.